src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesSignedCommits.java

```diff
@@ -113,8 +113,7 @@ public UsesSignedCommits update(ValueSet values) throws IOException {
     try {
       return new BooleanValue(USES_VERIFIED_SIGNED_COMMITS, askGithub());
     } catch (IOException e) {
-      System.out.printf("[x] Couldn't fetch data from api.github.com!%n");
-      System.out.printf("[x]     %s%n", e);
+      logger.error("Couldn't fetch data from api.github.com!", e);
```

Comment:
Hmm why logger.error only here and logger.warn at other places for the exception? :)

Reply:
For e.g., here https://github.com/SAP/fosstars-rating-core/pull/109/files#diff-93fae7ef1f1df793b699455448ed0af8R89

Reply:
There is no reason for that. That just happened and we didn't pay attention since it makes no harm. I'll update the code to use `warn()` to make it consistent (except the `main()` method).

Reply:
Cool thanks :)